strings/tips.txt

```diff
@@ -71,11 +95,12 @@ As the Chief Engineer, your hardsuit is significantly better than everybody else
 As the Chief Engineer, you can spy on and even forge PDA communications with the message monitor console! The key is in your office.
 As the Chief Engineer, your locker contains a jetpack upgrade that can be attached to hardsuits.
 As the Chief Engineer, the power flow control console in your office will show you APC infos and lets you control them remotely. 
-As an Engineer, the supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you.
+As an Engineer, the supermatter shard is an extremely dangerous piece of equipment: touching it will disintegrate you. So will touching it with telepathy.
 As an Engineer, you can electrify grilles by placing wire "nodes" beneath them: the big seemingly unconnected bulges from a half completed wiring job.
 As an Engineer, return to Engineering once in a while to check on the engine and SMES cells. It's always a good idea to make sure containment isn't compromised.
-As an Engineer, you can power the station solely with the solar arrays. While uninteresting, it is a much safer alternative to regular engines.
+As an Engineer, you can temporarily power the station solely with the solar arrays. They will provide just enough electricity to power the station for a while, however their output is just slightly below what is truely needed to consistently power the station.
 As an Engineer, you can cool the supermatter crystal by spraying it with a fire extinguisher. Only for the brave!
+As an Engineer, if you become face-to-face with the threat of a blob, try setting up a PA launcher (like the kind used in singularities) in an important chokehold. The accelerated particles will severely weaken and eventually destroy that line, allowing your crewmembers to push the offensive. This tactic is ineffective against blob cores themselves, however.
```

Comment:
No need to abbreviate PA here, just say particle accelerator. Actually I would rephrase this to be shorter, something like

"As an Engineer, the particle accelerator used to power the gravitational singularity is also effective at killing blobs."